### ISSUE TITLE
Move Table class variables to private

### ISSUE THREAD

**Issue by lordent** [kayak/pypika#120]
For example, this condition causes an AttributeError: 'bool' object has no attribute 'fields'
> .where(information_schema.table_name == 'public')

```
class Table(Selectable):
    def __init__(self, name, schema=None, alias=None):
        super(Table, self).__init__(alias)
        self.__table_name = name
        self.__schema = schema
```

**Comment by twheys** [comment]
This is a duplicate of https://github.com/kayak/pypika/issues/13 that I hadn't gotten around to earlier. Will fix this shortly.

**Comment by lordent** [comment]
@twheys thx, good job!

**Comment by twheys** [comment]
Published 0.14.2
